subprojects/core/src/main/java/org/gradle/util/Clock.java

```diff
@@ -16,11 +16,13 @@
 
 package org.gradle.util;
 
+import org.gradle.internal.time.DefaultEventTimer;
+
 /**
- * This class remains for backward compatibility. Replaced by `org.gradle.internal.time.Clock`.
+ * This class remains for backward compatibility. Replaced by `org.gradle.internal.time.DefaultEventTimer`.
  */
 @Deprecated
-public class Clock extends org.gradle.internal.time.Clock {
+public class Clock extends DefaultEventTimer {
```

Comment:
According to [the bible](https://wiki.eclipse.org/Evolving_Java-based_APIs_2#Evolving_API_Classes), this is safe.

This is a superclass expand/contract, which is safe as long as there are no references to the removed type or any of its methods.